tailwind.config.js

```diff
@@ -0,0 +1,32 @@
+/** @type {import('tailwindcss').Config} */
+
+const path = require('path');
+
+// Path to custom Tailwind plugins for Directory
+const dir = path.resolve(__dirname, 'src/tailwind/plugins');
+
+module.exports = {
+  prefix: 'su-',
+  presets: [
+    require('decanter'),
+  ],
+  content: [
+    /**
+     * We use GraphQL Typegen which generates a file at src/gatsby-types.d.ts
+     * Writing out the subdirectories in src/ here to prevent an infinite loop
+     * https://www.gatsbyjs.com/docs/how-to/styling/tailwind-css/#installing-and-configuring-tailwind
+     */
+    './src/pages/**/*.{js,jsx,ts,tsx}',
+    './src/components/**/*.{js,jsx,ts,tsx}',
+  ],
```

Comment:
Gatsby doc said this is how we should handle it if we have the TypeScript install. See URL in comment for info.

Reply:
Yep, this looks right to me.